spec/unit/pops/parser/lexer2_spec.rb

```diff
@@ -473,6 +473,21 @@ def epp_tokens_scanned_from(s)
         )
     end
 
+    it 'strips only last newline when using trim option' do
+      code = <<-CODE.unindent
+        @(END)
+        Line 1
+        
+        Line 2
+        -END
+        CODE
+      expect(tokens_scanned_from(code)).to match_tokens2(
+        [:HEREDOC, ''],
+        [:SUBLOCATE, ["Line 1\n", "\n", "Line 2\n"]],
+        [:STRING, "Line 1\n\nLine 2"],
+      )
+    end
+
```

Comment:
a test for the case when there is a newline after the last line of text is missing - test only tests that a single newline is removed.